src/ir/objc.rs

```diff
@@ -148,16 +165,17 @@ impl ObjCInterface {
     }
 }
 
-impl ObjCInstanceMethod {
-    fn new(name: &str, signature: FunctionSig) -> ObjCInstanceMethod {
+impl ObjCMethod {
+    fn new(name: &str, signature: FunctionSig, is_class: bool) -> ObjCMethod {
         let split_name: Vec<&str> = name.split(':').collect();
 
         let rust_name = split_name.join("_");
 
-        ObjCInstanceMethod {
+        ObjCMethod {
             name: name.to_owned(),
             rust_name: rust_name.to_owned(),
             signature: signature,
+            is_class: is_class,
```

Comment:
How do you feel about calling this and the accessor `is_class_method` instead, so it's a bit more explicit?